gui/src/components/JobRuns/StatusItem.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withStyles } from '@material-ui/core/styles'
+import Grid from '@material-ui/core/Grid'
+import ExpansionPanel from '@material-ui/core/ExpansionPanel'
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary'
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails'
+import Typography from '@material-ui/core/Typography'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+import StatusIcon from 'components/JobRuns/StatusIcon'
+import classNames from 'classnames'
+
+const styles = theme => {
+  return {
+    borderTop: {
+      borderTop: 'solid 1px',
+      borderTopColor: theme.palette.divider
+    },
+    item: {
+      position: 'relative',
+      paddingLeft: 50
```

Comment:
Wasn't it suggested to avoid this and use Material UI's spacings ?

Reply:
👍 This value is fixed and not related to the theme spacing. I'll add it to the theme though as a custom value.